config/hackernews-search/config.yaml

```diff
@@ -8,5 +8,8 @@ summary_default_language: "eng"
 summary_num_sentences: 3
 summary_num_results: 7
 
+summary_prompt_name: vectara-summary-ext-v1.4.0
+#summary_prompt_name: vectara-summary-ext-v1.2.0
```

Comment:
Should we keep this commented-out or remove it?

Reply:
good catch. removed!